lib/routes/catti/news.ts

```diff
@@ -0,0 +1,119 @@
+import { DataItem, Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+
+type NewsCategory = {
+    title: string;
+    description: string;
+};
+
+const NEWS_TYPES: Record<string, NewsCategory> = {
+    ggl: {
+        title: '通知公告',
+        description: 'CATTI 考试通知和公告',
+    },
+    ywdt: {
+        title: '要闻动态',
+        description: 'CATTI 考试要闻动态',
+    },
+    zxzc: {
+        title: '最新政策',
+        description: 'CATTI 考试最新政策',
+    },
+};
+
+const handler: Route['handler'] = async (context) => {
+    const category = context.req.param('category');
+    if (category === undefined || !NEWS_TYPES[category]) {
```

Comment:
Unnecessary check of `category === undefined` since 
https://github.com/DIYgod/RSSHub/blob/eaaa28bc8a9d2a5ae261bf1026572ba6e4a2d7d6/lib/routes/catti/news.ts#L94 defines category as a compulsory path parameter, user who doesn't provide `category` will be rejected by middleware with https://github.com/DIYgod/RSSHub/blob/55b6afeed45e910e5fedfd0d5c0750df89572be7/lib/errors/types/not-found.ts#L1